internal/operators/manager.go

```diff
@@ -0,0 +1,163 @@
+package operators
+
+import (
+	"encoding/json"
+
+	"github.com/openshift/assisted-service/internal/host"
+
+	"github.com/kelseyhightower/envconfig"
+
+	"github.com/openshift/assisted-service/internal/operators/lso"
+	"github.com/openshift/assisted-service/internal/operators/ocs"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/go-openapi/swag"
+	"github.com/openshift/assisted-service/internal/common"
+	"github.com/openshift/assisted-service/models"
+)
+
+// Manager is responsible for performing operations against additional operators
+type Manager struct {
+	log                logrus.FieldLogger
+	ocsValidatorConfig *ocs.Config
+	ocsValidator       ocs.OcsValidator
+}
+
+//go:generate mockgen -package=operators -destination=mock_operators_api.go . API
+type API interface {
+	// ValidateOCSRequirements validates OCS requirements
+	ValidateOCSRequirements(cluster *common.Cluster) bool
```

Comment:
if it is interface, why we have specific ocs functionality?

Reply:
it looks like all this code is mainly ocs code? doesn't we wanted to create interface?

Reply:
Yes, it's mainly OCS but this is only the first PR that lays ground work for introducing the generic API. With this PR all the operator-related logic will be enclosed within one package, which will help with further refactoring. 
Basically I didn't want to do all the changes in PR to make the change sets smaller.

Reply:
This can be generalized to: 
`ValidateRequirements(cluster *common.Cluster, operatorType models.OperatorType) bool`

and the caller will invoke it for OCS needs by:
`ValidateRequirements(cluster, models.OperatorTypeOcs)`

Reply:
The idea is for that method to be replaced with
`ValidateCluster(cluster *common.Cluster) ([]api.ValidationResult, error)`
when OCS is migrated to the generic API and there is no OCS-specific code in the `operators.Manager`.